Beta_Integrations/ProofpointThreatResponse/ProofpointThreatResponse_test.py

```diff
@@ -0,0 +1,208 @@
+            "name": "Abuse Disposition",
+            "value": "Unknown"
+        }
+    ],
+    "events": [
+        {
+            "id": 3,
+            "category": "malware",
+            "severity": "Info",
+            "source": "Proofpoint TAP",
+            "threatname": "",
+            "state": "Linked",
+            "description": "",
+            "attackDirection": "inbound",
+            "received": "2018-05-26T21:07:17Z",
+            "malwareName": "",
+            "emails": [
+                {
+                    "sender": {
+                        "email": "test"
+                    },
+                    "recipient": {
+                        "email": "test"
+                    },
+                    "subject": "test",
+                    "messageId": "test",
+                    "messageDeliveryTime": {
+                        "chronology": {
+                            "zone": {
+                                "id": "UTC"
+                            }
+                        },
+                        "millis": 1544640072000,
+                    },
+                    "abuseCopy": "false",
+                    "body": "test",
+                    'bodyType': "test",
+                    'headers': "test",
+                    'urls': "test"
+                }
+            ],
+        }
+    ],
+    "quarantine_results": [],
+    "successful_quarantines": 0,
+    "failed_quarantines": 0,
+    "pending_quarantines": 0
+}
+
+INCIDENT_FIELD_CONTEXT = [
+    {
+        "Attack_Vector": "Email"
+    },
+    {
+        "Classification": "Spam"
+    },
+    {
+        "Severity": "Critical"
+    },
+    {
+        "Abuse_Disposition": "Unknown"
+    }
+]
+
+INCIDENT_FIELD_INPUT = [
+    (MOCK_INCIDENT, INCIDENT_FIELD_CONTEXT)
+]
+
+
+@pytest.mark.parametrize('incident, answer', INCIDENT_FIELD_INPUT)
+def test_get_incident_field_context(incident, answer):
+    incident_field_values = get_incident_field_context(incident)
+    assert incident_field_values == answer
+
+
+EMAIL_RESULT = [
+    {
+        'sender': "test",
+        'recipient': "test",
+        'subject': "test",
+        'message_id': "test",
+        'message_delivery_time': 1544640072000,
+        'body': "test",
+        'body_type': "test",
+        'headers': "test",
+        'urls': "test"
+    }
+]
+
+EMAILS_CONTEXT_INPUT = [
+    (MOCK_INCIDENT['events'][0], EMAIL_RESULT)
+]
+
+
+@pytest.mark.parametrize('event, answer', EMAILS_CONTEXT_INPUT)
+def test_get_emails_context(event, answer):
+    emails_context = get_emails_context(event)
+    assert emails_context == answer
+
+#
+# demisto_args = {
+#     'limit': '1'
+# }
+
+
+# def test_list_incidents_command(mocker):
+#     mocker.patch.object(ProofpointThreatResponse, 'get_incidents_request', return_value=[MOCK_INCIDENT])
+#     mocker.patch.object(demisto, 'args', return_value=demisto_args)
+#
+#     list_incidents_result = list_incidents_command()
+#     assert list_incidents_result is None
```

Comment:
Is this test supposed to be deleted or refactored?

Reply:
Will be removed